src/test/cargo-rmc/simple-extern/src/externs.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+extern "C" {
+    pub fn call_into_rust(i: u32) -> u32;
+}
+
+#[no_mangle]
+pub extern "C" fn takes_int(i: u32) -> u32 {
```

Comment:
A cleaner name might mention that this is a exported rust function

Reply:
Fixed.